cobc/codegen.c

```diff
@@ -4298,12 +4298,7 @@ output_funcall_item (cb_tree x, const int i, unsigned int func_nolitcast)
 
 /* Use constant strings to replace string comparisons by more
  * efficient pointer comparisons */
-static const char *cob_prof_exit_paragraph_str = "cob_prof_exit_paragraph";
-static const char *cob_prof_exit_section_str = "cob_prof_exit_section";
-static const char *cob_prof_enter_paragraph_str = "cob_prof_enter_paragraph";
-static const char *cob_prof_enter_section_str = "cob_prof_enter_section";
-static const char *cob_prof_use_paragraph_entry_str = "cob_prof_use_paragraph_entry";
-static const char *cob_prof_stayin_paragraph_str = "cob_prof_stayin_paragraph";
+static const char *cob_prof_function_call_str = "cob_prof_function_call";
```

Comment:
Is this block still reasonable now - as we never use that outside of a marker - or can f->name just be `NULL` to cover that?

Reply:
I think using `NULL` would make the code less robust to future change. On the long term, we could create a new `cb_tree` node in `tree.h` for calls to profiling functions.

Reply:
OK